spec/spec_helper.rb

```diff
@@ -1,6 +1,11 @@
 $LOAD_PATH.unshift File.join(__dir__, '../lib')
 $LOAD_PATH.unshift File.join(__dir__, 'support')
 
+require 'openssl'
+
+$directory_url = ENV['ACME_DIRECTORY_URL'] || 'https://127.0.0.1/directory'
```

Comment:
Could this be a method instead of a global? It totally works but the mere existence of a global feels dirty to me. At least a global method can't be mutated.